### PR TITLE
Addition of log2

### PR DESCRIPTION
**Pull request by shibatch**
This patch adds log2 and log2f.

https://github.com/shibatch/sleef/issues/101

### CODE REVIEW

**Comment by fpetrogalli-arm** [comment]
Could you please also add the "finite" names testing? In https://github.com/shibatch/sleef/blob/master/src/libm-tester/finite_names_testing.c

They are not complete as most of the other functions are missing, but if you add the tests for `__log2[f]_finite` here in this PR, we won't forget it for sure.

**Comment by shibatch** [comment]
Since the tester only covers asin and acos at this point, I think it is better to make another PR for committing a patch to make it cover a lot more functions.
Please write an issue to explain what it is really going to test.

File: src/libm-tester/iutsimd.c [hunk 1]
```diff
@@ -416,12 +416,7 @@ int do_test(int argc, char **argv) {
     func_d_d("exp10", xexp10);
     func_d_d("expm1", xexpm1);
     func_d_d("log10", xlog10);
-    func_d_d("log1p", xlog1p);
```
Comment (fpetrogalli-arm) [inline src/libm-tester/iutsimd.c]: Could you please explain why you are removing these lines?

Comment (shibatch) [reply src/libm-tester/iutsimd.c]: These lines were accidentally duplicated in the previous code.
The same tests was executed twice, which was just waste of time.

**Pull request merged**